### PR TITLE
Add node3 to list of initial endpoints

### PR DESCRIPTION
**Pull request by octonato**

### CODE REVIEW

File: src/main/g8/src/main/resources/local-shared.conf [hunk 1]
```diff
@@ -21,12 +21,11 @@ akka.discovery.config.services {
   "$name;format="word-only,lowercase"$" {
     endpoints = [
       {host = "127.0.0.1", port = 9101},
-      {host = "127.0.0.1", port = 9102}
+      {host = "127.0.0.1", port = 9102},
+      {host = "127.0.0.1", port = 9103}
```
Comment (octonato) [inline src/main/g8/src/main/resources/local-shared.conf]: Not sure if that was an explicit choice, but node3 is missing. 

As a user, I may have `sbt -Dconfig.resource=local3.conf run` in my history and eventually start node3 as first one. Without it listed on the endpoints, it won't start the cluster.

File: src/main/g8/src/main/resources/local-shared.conf [hunk 2]
```diff
@@ -21,12 +21,11 @@ akka.discovery.config.services {
...
     ]
   }
 }
 
 # for reduced Projection latency
 akka.persistence.cassandra.events-by-tag.eventual-consistency-delay = 200 ms
-
-
```
Comment (octonato) [inline src/main/g8/src/main/resources/local-shared.conf:32]: No idea why this two empty lines were removed. Locally I see there is one blank like on lnum 32

**Review by patriknw — APPROVED** [review]
LGTM, pls update the code in akka-platform-guide also

File: src/main/g8/src/main/resources/local-shared.conf [hunk 3]
```diff
@@ -21,12 +21,11 @@ akka.discovery.config.services {
   "$name;format="word-only,lowercase"$" {
     endpoints = [
       {host = "127.0.0.1", port = 9101},
-      {host = "127.0.0.1", port = 9102}
+      {host = "127.0.0.1", port = 9102},
```
Comment (ennru) [inline src/main/g8/src/main/resources/local-shared.conf]: We could possibly remove the ending `,`

**Review by ennru — APPROVED** [review]

File: src/main/g8/src/main/resources/local-shared.conf (hunk 3 above)
Comment (octonato) [reply src/main/g8/src/main/resources/local-shared.conf]: done

**Pull request merged**